math/mathcore/inc/Fit/FitUtil.h

```diff
@@ -36,14 +36,14 @@
 #define USE_PARAMCACHE
 
 #ifdef R__HAS_VECCORE
-namespace vecCore{
+namespace vecCore {
 template <class T>
 vecCore::Mask_v<T> Int2Mask(unsigned i)
 {
    T x;
-   for (unsigned j = 0; j < vecCore::VectorSize<T>(); j++) vecCore::Set<T>(x, i, i + 1);
-   return vecCore::Mask_v<T>(x < i);
-   }
+   for (unsigned j = 0; j < vecCore::VectorSize<T>(); j++) vecCore::Set<T>(x, j, j + 1);
+   return vecCore::Mask_v<T>(x <= i);
+}
```

Comment:
Yes, I understood perfectly. I just used my weird formulation to map my use-case values :)

The same line loop was clang-format I think.

I'll change this